### PR TITLE
Add explicit outcomes in README

### PR DESCRIPTION
**Pull request by kelson42**
Attempt to better explain/document what Kiwix Build does and how to use it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -164,8 +169,43 @@ Kiwix-build.py will create several directories:
...
+
+Each time we use the CI/CD of Kiwix Build, it is checked if an
+appropriate base dependences tarball is available. If "yes" it is
+downloaded and used, otherwise it will be built and uploaded.
+
+If you need to create a new tarball you can either:
+* Increment the version number of them in `kiwixbuild/versions.py` a
+  new version is uploaded
+* Delete the multiple tarballs related to current version in
+  tmp.kiwix.org/ci.
```
Comment (mgautierfr) [inline README.md:192]: Preferably the first option. Deleting tarballs on ftp should be a last solution (and done only by us)

File: README.md [hunk 2]
```diff
@@ -164,8 +169,43 @@ Kiwix-build.py will create several directories:
...
+compiled dependences necessary to its compilation. These dependences
+can be third party or internal libraries (like libzim or libkiwix).
+
+They are available [here](http://tmp.kiwix.org/ci) and are built each
+night.
+
+#### Releases
+
+Kiwix Build is in charge of the release of binaries for many Kiwix
+project. Releases are triggered via git tagging.
```
Comment (mgautierfr) [inline README.md:206]: And nigthlies ?

**Review by mgautierfr — COMMENTED** [review]
As said IRL, it would be better to move all the CI information in a different file to not pollute the main README.md

Few more information:
- All based dependencies are build with release options (all the time)
- Project dependencies (other than base depedencies) are build in debug.
- Nigthlies are build in debug
- Release are build in release

**Comment by kelson42** [comment]
Move the CI/CD to https://github.com/kiwix/kiwix-build/wiki/Kiwix-Build-usage-in-CI-&-CD

**Pull request merged**